src/lib/sql/sql_translator.cpp

```diff
@@ -266,6 +277,26 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_select_statement(cons
   return _current_lqp;
 }
 
+void SQLTranslator::_translate_hsql_with_description(hsql::WithDescription& desc) {
+  SQLTranslator with_translator{_use_mvcc, nullptr, _parameter_id_allocator, _with_descriptions};
```

Comment:
did you test placeholders in WITH? `WITH a AS SELECT * FROM t WHERE a > ? SELECT ...`?

Reply:
Oh these placeholders are for prepared statements, right? Not yet.

Reply:
ping?

Reply:
ping ping

Reply:
Currently, I'm try to understand the placeholder implementation to create a test afterwards.